### ISSUE TITLE
Implement gene symbol list

### ISSUE THREAD

**Issue by tanyasg** [allen-cell-animated/cell-line-catalog#134]
Multi-edit gene symbols are currently stored like this:
![image](https://user-images.githubusercontent.com/25040762/123496572-93420c00-d5dd-11eb-83c8-387059461ea0.png)
 The `Main_gene_symbol` gets displayed in the cell line table view:
![image](https://user-images.githubusercontent.com/25040762/123496603-bf5d8d00-d5dd-11eb-9db3-350662122f3c.png)
and the `Main_gene_symbol1`, 2, and 3 are displayed here:
![image](https://user-images.githubusercontent.com/25040762/123496642-ea47e100-d5dd-11eb-8bf3-77d96bae93c2.png)

Goal is to remove the redundancy and convert `Main_gene_symbol` to store a list that gets used to populate both the entry in the cell line table view and the individual line view. New implementation should work for both single gene edit lines and multi-gene edit lines like the example above

**Comment by TravisKroeker** [comment]
What would be the impact of this change? How much work would it be? Would there be any front end changes? Should we maintain this ticket?

**Comment by meganrm** [comment]
Yeah, it would make storing data easier, ie each gene combination isn't a new thing, just a combination of existing genes. It would be small front end change, but I think we should do it for 2.0 because I already have ideas about storing the gene/protein info separately
